lib/sie_exporter.rb

```diff
@@ -0,0 +1,117 @@
+# encoding: utf-8
+
+require 'sie'
+
+# Exports activity years to SIE.
+class SieExporter
+  def initialize(activity_year)
+    @activity_year = activity_year
+  end
+
+  def call
+    Sie::Document.new(self).render
+  end
+
+  def program
+    'MAGE'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.